staging/src/k8s.io/apiserver/pkg/authentication/user/user.go

```diff
@@ -70,8 +70,10 @@ func (i *DefaultInfo) GetExtra() map[string][]string {
 const (
 	SystemPrivilegedGroup = "system:masters"
 	NodesGroup            = "system:nodes"
-	AllUnauthenticated    = "system:unauthenticated"
-	AllAuthenticated      = "system:authenticated"
+	// TODO(logicalhan): configure flowcontrol for requests from this group
+	MonitoringGroup    = "system:monitoring"
```

Comment:
I'm -1 on a built-in monitoring group ... that is not well-defined across clusters like nodes are